hibernate-core/src/main/java/org/hibernate/hql/internal/ast/tree/FromClause.java

```diff
@@ -61,8 +61,6 @@
 	 */
 	private List impliedElements = new LinkedList();
 
-	private List<EntityJoinFromElement> entityJoinFromElements;
```

Comment:
Now the above field has been redundant for it is included in `fromElements` which reserves join order via `LinkedHashSet`.